workspace/solution.go

```diff
@@ -12,8 +12,11 @@ import (
 	"github.com/exercism/cli/visibility"
 )
 
+const ignoreSubdir = ".exercism"
 const solutionFilename = ".solution.json"
 
+var solutionRelPath = filepath.Join(ignoreSubdir, solutionFilename)
```

Comment:
Seems like this is only used within NewSolution does it need to be a package level variable?

Reply:
Im speaking about solutionRelPath. I did a quick search and didn’t see other references. If that variable is used three times and we don’t plan on changing it for testing should it be const?

Reply:
It can't be const: `[gometalinter] const initializer filepath.Join(ignoreSubdir, solutionFilename) is not a constant (vet) [Error]`

Reply:
Sorry I wasn’t clear, if we make solutionFilename a const with the new path we wouldn’t need the other variables as they are just being used to construct the final solution path 
`const SolutionFilename = filepath.Join(“.exercism”,”solution.json”)`

I also think the it should be exported to denote that it is actually shared or is expected to be.

Reply:
Gotcha. I don't think `filepath.Join` can be const: `[gometalinter] const initializer filepath.Join(“.exercism”, ”solution.json”) is not a constant (vet) [Error]`

Reply:
Gotcha. It appears that `filepath.Join` can be const: `[gometalinter] const initializer filepath.Join(".exercism", "solution.json") is not a constant (vet) [Error]`

Reply:
Well then it seems like the best advise I can give is to carry on as you were. Disregarding my coments on this file. 

 I didnt think about the the separator not being known until run time making it unfit to use as a const, my fault. 

Thanks for vetting and for your time.

Reply:
No problem. I appreciate your feedback.